src/App.jsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react'
+import React from "react";
+import Authentication from "./components/Authentication";
+import MyPage from "./components/MyPage";
+import Header from "./components/Header";
+import { Route, Redirect } from "react-router-dom";
+import { useSelector } from "react-redux";
 
 const App = () => {
-  return (
-    <div>
-      
-    </div>
-  )
-}
+  const authenticated = useSelector((state) => state.authenticate);
+  const role = useSelector((state) => state.currentUser.role);
+
+  let display;
 
-export default App
+  if (authenticated && role === "journalist") {
+    display = <Redirect to="/journalist" />;
+  }
```

Comment:
Can you please an else here, what should happen if the logged in user is not a journalist? Can we redirect to a page that indicates for the registered user that he or she is in the wrong place?